ncoreparser/parser.py

```diff
@@ -25,24 +30,43 @@ def get_key(data):
         raise NcoreParserError(f"Error while read user "
                                f"key with pattern: {key_pattern}")
 
+
+    def id_exists(self, data, search_id):
+        for i, id in enumerate(data):
+            if search_id in data[i]:
+                return True
+
+        return False
+
+
     def get_items(self, data):
         types = self.type_pattern.findall(data)
-        ids_names = self.id_name_pattern.findall(data)
+        ids_names_posters = self.id_name_poster_pattern.findall(data)
+        ids_names = self.id_name_patter.findall(data)
         dates_times = self.date_pattern.findall(data)
         sizes = self.size_pattern.findall(data)
         seed = self.seeders_pattern.findall(data)
         leech = self.leechers_pattern.findall(data)
-        if len(types) != 0 and len(types) == len(ids_names) == \
+
+        # Not all torrents has poster img
+        if (len(ids_names) != len(ids_names_posters)):
+            for i, id in enumerate(ids_names):
+                if not self.id_exists(ids_names_posters, ids_names[i][0]):
+                    missing_torrent_data = (ids_names[i][0], ids_names[i][1], 'no cover')
```

Comment:
The no Cover could be a constant. Because in one place `no cover` in another `No cover`

Reply:
in `data.py` you can create a NO_COVER constant and use here.